frontend/src/component/onboarding/SelectSdk.tsx

```diff
@@ -0,0 +1,27 @@
+import { Box, styled, Typography } from '@mui/material';
+
+const SpacedContainer = styled('div')(({ theme }) => ({
```

Comment:
copied from the GenerateApiKey stage so may create common abstraction in another PR